### ISSUE TITLE
FlowField.detach() does not preserve Axes attribute

### ISSUE THREAD

**Issue by aschuh-hf** [BioMedIA/deepali#50]
The `FlowField.sitk()` function used by `FlowField.write()` uses `self.detach()` to detach the flow field from the computational graph. The detached flow field always has default `Axes.CUBE_CORNERS` even if the original flow field has `Axes.WORLD`. When the vectors are then mapped to `Axes.WORLD`, they are being modified. This results in an incorrect vector field to be saved.
